src/main.py

```
@@ -0,0 +1,51 @@
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+@app.get("/health")
+async def health_check():
+    """ヘルスチェックエンドポイント"""
+    response = {
+        "status": "healthy",
+        "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)